.github/workflows/build-images-releases.yaml

```diff
@@ -79,8 +87,7 @@ jobs:
           platforms: linux/amd64,linux/arm64
           tags: |
             ${{ github.repository_owner }}/${{ matrix.name }}:${{ steps.tag.outputs.tag }}
```

Comment:
What is this tag? Is it expected that we are not using env variable for this one?

Reply:
oh good catch, i think i just missed this one and the one below 🙀

Reply:
ah this tag is for dockerhub. let's keep it as is for this pull request. it would be a bit confusing to replace this with `${{ env.QUAY_ORGANIZATION }}`. if needed we can add a separate `DOCKERHUB_ORGANIZATION` environment variable and then update this line in a separate pull request.